integrations/anthropic/src/index.ts

```diff
@@ -41,13 +41,13 @@ export default new bp.Integration({
   actions: {
     generateContent: async ({ input, logger }) => {
       return await generateContent<ModelId>(<llm.GenerateContentInput>input, anthropic, logger, {
-        models,
+        models: languageModels,
         defaultModel: 'claude-3-5-sonnet-20240620',
       })
     },
-    listModels: async ({}) => {
+    listLanguageModels: async ({}) => {
```

Comment:
**note**: I already discussed this change with Yann, they aren't yet using the `listModels` action so this won't break anything.